src/components/users/Accounts.jsx

```diff
@@ -235,6 +262,69 @@ const CreateAccount = ({
     );
 };
 
+const RootAccount = ({
+    idPrefix,
+    passwordPolicy,
+    setIsRootValid,
+    accounts,
+    setAccounts,
+}) => {
+    const [password, setPassword] = useState(accounts.rootPassword);
+    const [confirmPassword, setConfirmPassword] = useState(accounts.rootConfirmPassword);
+    const [isPasswordValid, setIsPasswordValid] = useState(false);
+    const isRootAccountEnabled = accounts.isRootEnabled;
+
+    useEffect(() => {
+        setIsRootValid(isPasswordValid || !isRootAccountEnabled);
+    }, [setIsRootValid, isPasswordValid, isRootAccountEnabled]);
+
+    const rootAccountCheckbox = content => (
+        <Checkbox
+          id={idPrefix + "-enable-root-account"}
+          label={_("Enable root account")}
+          isChecked={isRootAccountEnabled}
+          onChange={(_event, enable) => setAccounts(ac => ({ ...ac, isRootEnabled: enable }))}
+          body={content}
+        />
+    );
+
+    const passphraseForm = (
+        <PasswordFormFields
+          idPrefix={idPrefix}
+          policy={passwordPolicy}
+          password={password}
+          setPassword={setPassword}
+          passwordLabel={_("Passphrase")}
+          confirmPassword={confirmPassword}
+          setConfirmPassword={setConfirmPassword}
+          confirmPasswordLabel={_("Confirm passphrase")}
+          rules={[]}
+          setIsValid={setIsPasswordValid}
+        />
+    );
+
+    useEffect(() => {
+        setAccounts(ac => ({ ...ac, rootPassword: password, rootConfirmPassword: confirmPassword }));
+    }, [setAccounts, password, confirmPassword]);
+
+    return (
+        <Form
+          isHorizontal
+          id={idPrefix}
+        >
+            <Title
```

Comment:
I think the propper approach would be to user FormSection component. https://www.patternfly.org/components/forms/form#form-sections

Reply:
Used, it produces slightly different layout in tests (spacing of the sections and title sizes) but for me it is acceptable.